src/main/java/practice/StreamPractice.java

```diff
@@ -50,7 +74,15 @@ public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toA
      */
     public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<Person> peopleList) {
-        return Collections.emptyList();
+        Predicate<Person> filterPersonByAge = person -> person.getSex().equals(Person.Sex.WOMAN)
```

Comment:
```suggestion
        Predicate<Person> filterPersonByAge = person -> person.getSex() == Person.Sex.WOMAN
```
we should compare enums using ==